test/unit/lib/session.js

```diff
@@ -215,40 +214,41 @@ describe('Keystone.session', function () {
 
 			});
 
-			it('shoud match email with all uppercase', function (done) {
-				var lookup = { email: 'TEST@TEST.COM', password: 'password'};
-					keystone.session.signin(lookup, null, null, function () {
-					// make sure .findOne() is called with a regular expression
-					sinon.assert.calledOnce(this.User.model.findOne);
-					this.User.model.findOne.getCall(0).args[0].email.must.be.instanceof(RegExp);
-					// make sure .exec() is called after
-					sinon.assert.calledOnce(this.User.model.exec);
-					this.User.model.exec.calledAfter(this.User.model.findOne).must.be.true;
-					// make sure .signinWithUser() is called on successful match
-					sinon.assert.calledOnce(keystone.session.signinWithUser);
-					done();
-				}.bind(this), this.onFailure);
-
-			});
-
-			it('shoud match email with all lowercase', function (done) {
-				var lookup = { email: 'test@test.com', password: 'password'};
-				keystone.session.signin(lookup, null, null, function () {
-					// make sure .findOne() is called with a regular expression
-					sinon.assert.calledOnce(this.User.model.findOne);
-					this.User.model.findOne.getCall(0).args[0].email.must.be.instanceof(RegExp);
-					// make sure .exec() is called after
-					sinon.assert.calledOnce(this.User.model.exec);
-					this.User.model.exec.calledAfter(this.User.model.findOne).must.be.true;
-					// make sure .signinWithUser() is called on successful match
-					sinon.assert.calledOnce(keystone.session.signinWithUser);
-					done();
-				}.bind(this), this.onFailure);
-
-			});
+			// it('should match email with all uppercase', function (done) {
```

Comment:
Similar to earlier comment: would either note why these tests are disabled or remove if they are unlikely to be relevant in future.